pkgs/applications/science/biology/sratoolkit/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, lib
+, fetchurl
+, autoPatchelfHook
+, libidn
+, zlib
+, bzip2
+}:
+
+stdenv.mkDerivation rec {
+  name = "sratoolkit";
+
+  version = "2.11.3";
+
+  libidn11 = libidn.overrideAttrs (old: {
+    name = "libidn-1.34";
+    src = fetchurl {
+      url = "mirror://gnu/libidn/libidn-1.34.tar.gz";
+      sha256 = "0g3fzypp0xjcgr90c5cyj57apx1cmy0c6y9lvw2qdcigbyby469p";
+    };
+  });
+
+  src = fetchurl {
+    url = "https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/${version}/sratoolkit.${version}-ubuntu64.tar.gz";
+    sha256 = "1590lc4cplxr3lhjqci8fjncy67imn2h14qd2l87chmhjh243qvx";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    libidn11
+    zlib
+    bzip2
+    stdenv.cc.cc.lib
+  ];
+
+  sourceRoot = "./sratoolkit.${version}-ubuntu64/bin";
+
+  installPhase = ''
+    for i in $(ls -p | grep -v / | grep -v remote-fuser ) ;  do
```

```suggestion
    for i in $(ls -p | grep -v / | grep -v remote-fuser);  do
```
Please do not rely on ls. How many files do we need to copy? Maybe just do it manually.

---

Please review the new version, I don't know if this is really better. The new version has ~100 lines more.

---

Thats not really better. Can we use find to search for executables?

---

Yes, looks better.